src/main/game/entities/EntityId.java

```diff
@@ -28,4 +28,13 @@ public Enum getSubTypeId(){
     public int getInstanceId(){
         return this.instanceId;
     }
+
+    //If entityId equals, return 1, else return 0;
+    public int compareTo(EntityId entityId){
+        if(this.playerId == entityId.getPlayerId()&&this.instanceId==entityId.getInstanceId()
```

Comment:
Also, you don't need an else statement if you're just returning in both cases, but that's up to you